jsonpickle/util.py

```diff
@@ -43,10 +43,19 @@ def is_type(obj):
     >>> is_type(Klass)
     True
     """
+    # use "isinstance" and not "is" to allow for metaclasses
     if PY3:
-        return type(obj) is type
+        return isinstance(obj, type)
     else:
-        return type(obj) is type or type(obj) is types.ClassType
+        return isinstance(obj, (type, types.ClassType))
+
+
+def has_method(obj, name, *args, **kwargs):
+    try:
+        getattr(obj, name)(*args, **kwargs)
+        return True
+    except:
+        return False
```

Comment:
@davvid Alright then, I'll do it :)

Another minor thing I've noticed -- have you considered replacing the boilerplate in `jsonpickle.compat` with `six` so you don't have to reinvent the wheel? (as recommended in the official python docs --https://docs.python.org/3/howto/pyporting.html#drop-support-for-python-2-5-and-older-at-least). It's a slim requirement and the chances are most users already have it installed due to it being a very common dependency.

Otherwise, we'll have to add even more unneeded copy/paste style boilerplate to `compat` in order to have proper metaclass unit tests in both python 2/3... It also provides things like `six.class_types`, `six.get_unbound_method` which are done differently in 2/3 and could be handy here.